iot/api-client/beta-features/commands/receive/receive_test.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Nit: Google LLC

Drop the "All rights reserved."

Reply:
Done.